test/unit/queue/src/CollectiveQueue.cpp

```diff
@@ -25,7 +25,7 @@
 struct QueueCollectiveTestKernel
 {
     template<typename TAcc>
-    ALPAKA_FN_ACC auto operator()(
+    auto operator()(
```

Comment:
Is this change related to the CUDA 10.2 support?

Reply:
Yes. This fixes a host-device warning that gets an error with the 10.2 werror setting.